### PR TITLE
In App Messaging for iOS & New Example App

### PR DESCRIPTION
**Pull request by rgomezp**
* iOS bridging for native version 2.11.0
* New example app created using React Native version 0.60.5
   - uses auto-linking
   - uses cocoapods

### CODE REVIEW

File: ios/RCTOneSignal/RCTOneSignalEventEmitter.m [hunk 1]
```diff
@@ -347,11 +347,11 @@ + (void)sendEventWithName:(NSString *)name withBody:(NSDictionary *)body {
 }
 
 RCT_EXPORT_METHOD(setExternalUserId:(NSString *)externalId) {
-    [OneSignal setExternalUserId:externalId];
+    //[OneSignal setExternalUserId:externalId];
```
Comment (mikechoch) [inline ios/RCTOneSignal/RCTOneSignalEventEmitter.m]: uncomment now, code exists

File: ios/RCTOneSignal/RCTOneSignalEventEmitter.m [hunk 2]
```diff
@@ -347,11 +347,11 @@ + (void)sendEventWithName:(NSString *)name withBody:(NSDictionary *)body {
...
 }
 
 RCT_EXPORT_METHOD(removeExternalUserId) {
-    [OneSignal removeExternalUserId];
+    //[OneSignal removeExternalUserId];
```
Comment (mikechoch) [inline ios/RCTOneSignal/RCTOneSignalEventEmitter.m]: uncomment now, code exists

File: ios/RCTOneSignal/RCTOneSignalEventEmitter.m [hunk 3]
```diff
@@ -361,4 +361,49 @@ + (void)sendEventWithName:(NSString *)name withBody:(NSDictionary *)body {
 RCT_EXPORT_METHOD(initInAppMessageClickHandlerParams) {
     //unimplemented in iOS
 }
+
+RCT_EXPORT_METHOD(addTriggers: (NSDictionary *)triggers) {
```
Comment (Nightsd01) [inline ios/RCTOneSignal/RCTOneSignalEventEmitter.m]: Nit: what’s with the parameter spacing on the exported methods? Would suggest removing the spaces

**Review by Nightsd01 — APPROVED** [review]
LGTM asides from a small syntax nit!

**Review by mikechoch — APPROVED** [review]

**Review by mikechoch — APPROVED** [review]

**Pull request merged**